packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -730,7 +730,12 @@ function ArrayField({
     setSelectedBadgeIndex(undefined);
   };
   const addItem = async () => {
-    if ((!!currentFieldValue || currentFieldValue === false) && !hasError) {
+    if (
+      (currentFieldValue !== undefined ||
+        currentFieldValue !== null ||
+        currentFieldValue !== \\"\\") &&
```

Comment:
we can use double not to check for falsy values

Reply:
I want to allow a false here. It was a double not previously.

Reply:
I want to allow a number 0 as well.